docs/content/guides/dapp-deployment/mainnet-deployment.md

```diff
@@ -0,0 +1,70 @@
+
+Breaking changes will be a fact of life for early adopters of Flow's development stack, and you may often need to manually re-deploy your application's contracts after updates. But that's OK we're here to help!
+
+### Version Compatibility
+
+A version compatibility table can be found here. This will help you navigate version compatibility between Emulator, SDK, and Network Node (flow-go) versions.
+
+### Mainnet Sporking
+
+"Sporking" (soft forking) is the process of upgrading Flow network node software, and migrating the state from the previous version.
+
+Currently, **historical event data is not migrated between sporks,** so you'll need to design your application with this in mind**.** We recognize the usefulness of historical event data and plan on adding a means of accessing it in the near future.
+
+### "Real Value" on Mainnet
+
+Unlike Flow Testnet, once your application is launched on Flow Mainnet, your users will be exchanging real value in the form of Flow tokens (The Flow blockchain's native token) or other resources which have been made available. You or your user will also be responsible for paying transaction fees using Flow tokens.
+
+You can read about Transaction fees here: [https://docs.onflow.org/token/concepts/#fees](https://docs.onflow.org/token/concepts/#fees)
+
+### Ongoing Development
+
+The goal of the Flow project is to become decentralized and community operated, over time. In the early stages of the network the Flow core team will be primarily responsible for the stability of the network, adding protocol features included in the node software...etc. The stability of the network will increase over time, as more nodes join and begin to participate in the protocol, however there are still "unknown-unknowns" (an apt idiom from a regrettable source) Early adopters of Flow are test pilots attempting to break blockchain barriers of the past, and we're asking all early-adopters for their help flagging issues with the Flow core team.
+
+## Deploying on Mainnet
+
+Currently, deploying to Mainnet requires manual intervention from the Flow team. To make your deployment as fast and as smooth as possible please use the steps in the guide to prepare your project.
+
+### Prerequisites
+
+Before considering your application for deployment to Mainnet, it has to have been live on Testnet handling real world traffic for a fixed period of time as suggested here: **Testnet Testing Guidelines.**
+
+Your project's smart contracts should include test coverage and follow any additional guidelines set out here: [Smart Contract Testing Guidelines](../contract-testing).
+
+Your reviewers will be using the following criterion when assessing your project: [Flow Project Technical Analysis](../contract-testing) Keeping these points in mind while developing your project will help speed up the review and deployment process**.**
```

Comment:
Why is this period bold?

Reply:
Just want to make sure folks know that sentence is finished...